ui/hooks/experiences/useBridging.ts

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react';
+import { useDispatch } from 'react-redux';
+import { fetchBridgeFeatureFlags } from '../../pages/bridge/bridge.util';
+import { setBridgeFeatureFlags } from '../../store/actions';
+
+const useBridging = () => {
```

Comment:
Should we handle this on initialization of the bridge state (i.e., when the experience loads) instead?

Reply:
I considered that, but went with this approach bc we need the FF to determine the Bridge button's `onClick` behavior. When the feature is fully released though, I think we can do this

Reply:
That makes sense